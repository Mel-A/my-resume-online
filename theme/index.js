const theme = {
  colors: {
    alabaster: '#FAFAFA',
    astral: '#327DA0',
    aquamarine: '#A1DAD7',
    azure: '#0080FF',
    cobalt: '#003DB3',
    gray: '#333333',
    white: '#FFFFFF'
  },
  fonts: {
    small0: '10px',
    small8: '18px',
    medium0: '20px',
    medium4: '24px',
    big0: '30px'
  }
}

export default theme