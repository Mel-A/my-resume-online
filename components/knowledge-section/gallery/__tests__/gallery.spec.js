import { shallow, mount } from 'enzyme'
import Gallery from '..'
import { ThemeProvider } from "styled-components"
import theme from '../../../../theme'

describe('gallery', () => {
it('gallery renders correctly', () => {
    const gallery = shallow(<Gallery />)
    expect(gallery).toMatchSnapshot()
  })

  it('gallery renders theme correctly', () => {
    const gallery = mount(<Gallery />, {
      wrappingComponent: ThemeProvider,
      wrappingComponentProps: {
        theme,
      },
    })
    expect(gallery).toMatchSnapshot()
  })
})
