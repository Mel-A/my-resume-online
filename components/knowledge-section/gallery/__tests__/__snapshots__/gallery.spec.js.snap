// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gallery gallery renders correctly 1`] = `ShallowWrapper {}`;
